src/RahaFirestore.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * A module that exports an api for interacting with Raha's Firestore db.
+ *
+ * Example usage:
+ * async function getAppliedOperations() {
+ *     return await get(
+ *         operationsCollectionFilters.applied(true)(
+ *             operationsCollection()));
+ * }
+ */
+
+import * as firebase from 'firebase';
+import 'firebase/firestore';
+
+/**
+ * Retrieve an instance of the Firestore db.
+ */
+function getFirebaseDb() {
+    return firebase.initializeApp(require('./config/firebase.config.json')).firestore();
+}
+
+/**
+ * Resolve a query on a Firestore collection.
+ */
+async function get(collection: firebase.firestore.CollectionReference) {
+    return (await collection.get()).docs;
+}
+
+/**
+ * Return the Operations collection.
+ */
+function operationsCollection() {
+    return getFirebaseDb().collection('operations');
+}
+
+/**
+ * Filters that can be applied to an Firestore collection.
+ */
+const operationsCollectionFilters = {
+    applied: (isApplied) => (collection) => collection.where('applied', '==', isApplied),
+};
+
+export {
+    get,
+    operationsCollectionFilters,
+    operationsCollection,
+};
```

Comment:
newline

Reply:
aside: we really should just add [`prettier`](https://github.com/prettier/prettier) and set up our editors to auto apply it/precommit hook to do that.

Reply:
I'm down, could you write up a quick readme on how to do that?